docs/04-configuration.md

```diff
@@ -23,7 +23,7 @@ Snowpack supports configuration files in multiple formats. Snowpack will look fo
 
 ### Configuration Options
 
-```json
+```js
```

Comment:
I think I forgot to change this back last time (for the comments)

Reply:
Also, I debated adding it here to Configuration Options, but I’m on the fence. I want this to clearly be “not recommended / only necessary if you’re absolutely certain” advanced config… should we add it here for completeness?

Reply:
I think it's fair to mention here, maybe with a link to your larger section below. Especially if this is going to be normal behavior for Svelte, this shouldn't be made to feel "unsupported/experimental".